wiki/Home.md

```diff
@@ -0,0 +1,26 @@
+
+Welcome to the unified-codes wiki!
+
+"unified-codes" is intended to complately replace the original universal drug codes server. 
+* The "original" universal drug codes server is currently located at https://sussol.github.io/universal-drug-codes-server/.
+* "unified-codes" (aka V2) is not yet deployed.
```

Comment:
Not sure how others feel, but I think a design principle for the wiki should be that it is as "user-friendly" as possible, e.g. something we can show to stakeholders/funders. In that respect, I think mentions of "unified-codes" versus "universal codes drug server" could cause quite a lot of confusion for users. IMO we can just differentiate on the basis of `v1` and `v2`, and would be good to keep discussion of `v1` to the legacy API documentation. 

If others agree, my suggestion would be something like this (but you can probably write it better!)

```
Welcome to the unified-codes wiki!

[Some info on unified-codes, e.g. "This wiki is intended as the main reference document/information hub for the unified-codes project"].

Note: the old `v1` universal codes server has been retired, but can still be found [here](https://sussol.github.io/universal-drug-codes-server/).
```

Reply:
Yep, agree with being "user-friendly" (though what dev consumers find friendly may be quite different to stakeholders 😆 ). I think splitting it into sections as you've suggested below will help with this distinction, and we can keep these overview type sections more focused on a non-technical audience. Also see how unified-codes vs universal codes is kind of confusing - I thought it may just help people looking for the implementation details to know it also has another name (but we can put the details in a separate section).

Reply:
Sorry, should have clarified! When I say user-friendly, I mean for general users of the services (e.g. avoid language that won't be understood by the average user). I think a good general pattern for documentation is to keep things simple at the top-level and leave implementation/technical details for more specific sections (e.g. "contributors", "technical reference", "advanced usage" etc.) Just opinion based on my experience!

Reply:
Yeah, that makes sense :). I'm not used to putting everything in one place - my pattern of thinking was separate docs/locations for different audiences but what you've described sounds like it would work well for the wiki!